config/PropertyNames.xml

```diff
@@ -278,11 +278,14 @@
         <property name="Registry.DynamicRegistration" languages="cpp" />
         <property name="Registry.Internal" class="ObjectAdapter" languages="cpp" />
         <property name="Registry.LMDB" class="LMDB" languages="cpp"/>
+        <property name="Registry.NodeSessionTimeout" languages="cpp" default="30" />
         <property name="Registry.PermissionsVerifier" class="Proxy" languages="cpp" />
         <property name="Registry.ReplicaName" languages="cpp" default="Master" />
+        <property name="Registry.ReplicaSessionTimeout" languages="cpp" default="30" />
         <property name="Registry.Server" class="ObjectAdapter" languages="cpp" />
         <property name="Registry.SessionFilters" languages="cpp" default="0" />
         <property name="Registry.SessionManager" class="ObjectAdapter" languages="cpp" />
+        <property name="Registry.SessionTimeout" languages="cpp" default="30" />
```

Comment:
The previous settings were 60. I assume we want 30 now?

Reply:
30 is what we document.

I will remove Registry.SessionTimeout in a follow-up PR so it's a moot point.